src/docs/src/conf.py

```diff
@@ -29,9 +30,13 @@
 
 nitpicky = True
 
-# should be over-written using rebar-inherited settings
-version = "3.3"
-release = "3.3.0"
+# load version numbers from version.mk
+version_file = Path().absolute().joinpath('../../../version.mk').resolve()
+with open(version_file) as file:
+    props = dict(line.strip().split('=', 1) for line in file)
+
+version = f"{props['vsn_major']}.{props['vsn_minor']}" #"3.3"
+release = f"{props['vsn_major']}.{props['vsn_minor']}.{props['vsn_patch']}" #"3.3.3"
```

Comment:
Do you want to leave the comments in place, which will become obsolete as soon as there's a new release?

Reply:
I have removed it. In the beginning it was meant to be easier to read, but it might just lead to confusion.